gradle/execution.gradle

```diff
@@ -1,37 +1,49 @@
-task downloadPbf << {
-    mkdir "${buildDir}/example/data/pbfSource/"
-    ant.get(src: "https://apple.box.com/shared/static/1wj4fiuwg88dczgkwxrva8udbbqjn6y5.pbf",
-        dest: "${buildDir}/example/data/pbfSource/belize.osm.pbf",
-        skipexisting: 'true')
+task downloadPbf {
+    doLast {
+        mkdir "${buildDir}/example/data/pbfSource/"
+        ant.get(src: "https://apple.box.com/shared/static/1wj4fiuwg88dczgkwxrva8udbbqjn6y5.pbf",
+            dest: "${buildDir}/example/data/pbfSource/belize.osm.pbf",
+            skipexisting: 'true')
+    }
 }
 
-task downloadBoundaries << {
-    mkdir "${buildDir}/example/data/"
-    ant.get(src: "https://apple.box.com/shared/static/k06amjcm32p83klwd5roex3m8vxj0mvx.gz",
-        dest: "${buildDir}/example/data/boundaries.txt.gz",
-        skipexisting: 'true')
+task downloadBoundaries {
+    doLast {
+        mkdir "${buildDir}/example/data/"
+        ant.get(src: "https://apple.box.com/shared/static/k06amjcm32p83klwd5roex3m8vxj0mvx.gz",
+            dest: "${buildDir}/example/data/boundaries.txt.gz",
+            skipexisting: 'true')
+    }
 }
 
-task downloadShardingTree << {
-    mkdir "${buildDir}/example/data/"
-    ant.get(src: "https://apple.box.com/shared/static/2p2nz3khyusaup3mdxmuyr4mifwx1bmx.txt",
-        dest: "${buildDir}/example/data/sharding.txt",
-        skipexisting: 'true')
+task downloadShardingTree {
+    doLast {
+        mkdir "${buildDir}/example/data/"
+        ant.get(src: "https://apple.box.com/shared/static/2p2nz3khyusaup3mdxmuyr4mifwx1bmx.txt",
+            dest: "${buildDir}/example/data/sharding.txt",
+            skipexisting: 'true')
+    }
 }
 
-task copyShard1 (dependsOn: 'downloadPbf') << {
+task copyShard1 (dependsOn: 'downloadPbf') {
+    doLast {
```

Comment:
what's with the new doLast for these tasks?

Reply:
The `<<` above used to be a shortcut for doLast. Now with gradle 5 this is gone.